lib/ipmi_backend_utils.pm

```diff
@@ -17,6 +17,7 @@ use Exporter;
 use strict;
 use warnings;
 use testapi;
+use version_utils qw/sle_version_at_least/;
```

Comment:
I think our perlcritic style dictates single ticks for single word imports. Also, I think the behaviour is not SLE version specific. I think `is_storage_ng` should work